src/client/pydaos/pydaos_shim.c

```diff
@@ -446,6 +446,273 @@ __shim_handle__cont_close(PyObject *self, PyObject *args)
 	return PyInt_FromLong(rc);
 }
 
+#define ITER_NR		96
+
+static int
+oit_mark(daos_handle_t oh, daos_handle_t oit)
+{
+	char			*buf = NULL;
+	daos_anchor_t		anchor = {0};
+	daos_key_desc_t		kds[ITER_NR];
+	d_sg_list_t		sgl;
+	d_iov_t			sg_iov;
+	daos_size_t		buf_size = ITER_NR * 256;
+	struct pydaos_df	entry;
+	size_t			size = sizeof(entry);
+	d_iov_t			marker;
+	bool			mark_data = true;
+	int			rc = 0;
+
+	D_ALLOC(buf, buf_size);
+	if (buf == NULL)
+		goto out;
```

Comment:
This needs to set rc.

Reply:
will address this with your other comments in a follow on patch.